apple/watchos.bzl

```diff
@@ -115,3 +116,30 @@ watchos_build_test(
 """,
     platform_type = "watchos",
 )
+
+def watchos_static_framework(name, **kwargs):
+    # buildifier: disable=function-docstring-args
+    """Builds and bundles a watchOS static framework for third-party distribution."""
+    avoid_deps = kwargs.get("avoid_deps")
+    deps = kwargs.get("deps")
+    apple_static_library_name = "%s.apple_static_library" % name
+
+    native.apple_static_library(
```

Comment:
I think we now have a starlark implementation of `apple_static_library` ?

Reply:
Where do I find that? I only see references to `native.apple_static_library` in `tvos.bzl` and `ios.bzl`

Reply:
🤦🏻  I was thinking of `apple_binary`, sorry for the confusion